rules/execute-command.js

```diff
@@ -101,13 +105,12 @@ function runCommand(command, args = []) {
     const cmd = command
       .replace(/\$\{NODE\}/g, exe)
       .replace(/\$\{ROOT_DIR\}/g, rootDir);
-    execOrExit(cmd, options);
+    await spawnOrExit(cmd, [], options);
   } else if (command in rootScripts) {
     // if command exists at root level but not at project level, run the root level command instead of erroring
-    execOrExit(`${node} ${yarn} run ${command} ${params}`, {
+    await spawnOrExit(`${node} ${yarn} run ${command} ${params}`, [], {
```

Comment:
Can you pass arguments to spawn like this?

Reply:
Yup, this is possible using `shell: true` option: https://github.com/uber-web/jazelle/blob/22e5853ead0ad8325f7c21c105a60d985688adb6/rules/execute-command.js#L90

It's technically only needs to use shell in this case: https://github.com/uber-web/jazelle/blob/22e5853ead0ad8325f7c21c105a60d985688adb6/rules/execute-command.js#L97-L108, where we take `command` as "granted" (i.e., user input may contain something that requires shell to interpret it). In other cases I could probably change it to use plain args

Reply:
👍